src/components/Discover/TrendingTokens.tsx

```diff
@@ -671,27 +725,40 @@ export function TrendingTokens() {
           style={{ marginHorizontal: -padding }}
         >
           <CategoryFilterButton
-            category={TrendingCategory.Trending}
+            category={categories[0]}
```

Comment:
Not blocking but the way `categories` is set up seems error prone to me. I think this should be more explicit.

Reply:
Yep I agree. Going to rework this later on